src/js/components/Paragraph/Paragraph.js

```diff
@@ -21,9 +23,9 @@ const Paragraph = forwardRef(
     return (
       <StyledParagraph
         ref={ref}
-        colorProp={color}
-        fillProp={fill}
-        size={size}
+        $colorProp={color}
```

Comment:
Since we have `Omit<JSX.IntrinsicElements['p'], 'color'> ` in the index.d.ts file for Paragraph I think we can use the name `$color` instead of `$colorProp`

Reply:
For some reason, still getting this flag: 

<img width="1121" alt="Screen Shot 2023-07-12 at 10 22 43 AM" src="https://github.com/grommet/grommet/assets/12522275/4d45666a-ca11-4551-8e48-550531e10b95">

Reply:
Ah weird, okay with moving forward as it